Analysis/Tasks/SkimmingTutorials/DataModel/UDDerived.h

```diff
@@ -0,0 +1,42 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+/// \author Nima Zardoshti <nima.zardoshti@cern.ch>, CERN
+
+#ifndef O2_ANALYSIS_UDDERIVED_H
+#define O2_ANALYSIS_UDDERIVED_H
+
+#include "Framework/ASoA.h"
+#include "Framework/AnalysisDataModel.h"
+
+namespace o2::aod
+{
+
+namespace udtrack
```

Comment:
I wonder if this is actually a udevent? Because it contains the two tracks inside. But it is a detail, no need to adapt in this PR

Reply:
This is true. The reason I called it track is that the skim would need to be evaluated with respect to the track table